lib/mountain_view/engine.rb

```diff
@@ -12,5 +16,11 @@ class Engine < ::Rails::Engine
         append_view_path app.root.join("app", "components")
       end
     end
+
+    initializer "mountain_view.add_helpers" do |app|
```

Comment:
Unused block argument - `app`. You can omit the argument if you don't care about it.